README.md

```diff
@@ -5,6 +5,9 @@
   <a href="https://github.com/bytebeamio/rumqtt/actions/workflows/main.yml">
     <img alt="build status" src="https://github.com/bytebeamio/rumqtt/actions/workflows/main.yml/badge.svg">
   </a>
+  <a href='https://coveralls.io/github/bytebeamio/rumqtt?branch=main'>
+    <img src='https://coveralls.io/repos/github/bytebeamio/rumqtt/badge.svg?branch=main' alt='Coverage Status' />
+  </a>
```

Comment:
```suggestion
  <a href="https://coveralls.io/github/bytebeamio/rumqtt?branch=main">
    <img alt="coverage status" src="https://coveralls.io/repos/github/bytebeamio/rumqtt/badge.svg?branch=main">
  </a>
```

also is explicit branch `main` required? doesn't it default to it anyways.

Reply:
This is what coveralls suggests, to include the name of the branch.